src/ResourceManager/Common/Commands.ResourceManager.Common/AzureRMCmdlet.cs

```diff
@@ -102,7 +102,7 @@ protected override IAzureContext DefaultContext
             {
                 if (DefaultProfile == null || DefaultProfile.DefaultContext == null || DefaultProfile.DefaultContext.Account == null)
                 {
-                    throw new PSInvalidOperationException("Run Login-AzureRmAccount to login.");
+                    throw new PSInvalidOperationException("Run Connect-AzureRmAccount to login.");
```

Comment:
Please move this into a resource string

Reply:
Or, better yet, use one of the resource strings below

Reply:
Done.  Neither of the resource strings below directly apply so created a new one.